modules/vba_documents/spec/factories/upload_submissions.rb

```diff
@@ -5,12 +5,12 @@
     source: nil, doc_type: 'Unknown', total_documents: 2, total_pages: 2,
     content: {
       page_count: 1,
-      dimensions: { height: 8.5, width: 11.0, oversized_pdf: false },
+      dimensions: { height: 11.0, width: 8.5, oversized_pdf: false },
```

Comment:
Wow, there it is. I remember you asking if they might have swapped the height width values and here they are, explicitly labeled. I guess in could have been a landscape mode document? 🤷